templates/grafana-operator.yaml

```diff
@@ -8,6 +8,8 @@ spec:
   selector:
     matchLabels:
       name: {{ .GrafanaOperatorName }}
+    matchLabels:
```

Comment:
I think it's ok for now to add extra selectors directly to the CR, but we might at some point look into passing them down from the `ApplicationMonitoring` CR: https://github.com/integr8ly/installation/blob/master/roles/middleware_monitoring/templates/application_monitoring_cr.yml.j2#L6